utils/signer/Cargo.toml

```diff
@@ -0,0 +1,25 @@
+[package]
```

Comment:
nit: IMO it might be a bit too granular to have a crate just for signer utils, given that validity and fault-proof are the only consumers. Maybe this stuff could just live in utils/host.

Reply:
I agree that it's a bit granular, but IMO utils/host is not a proper place for signer code to live in.

In the long term, I think there's a lot of room to refactor the code with shared utils such as signer across validity proposer and fault-proof proposer&challenger.

So I'm thinking of keeping as is, but in the future maybe renaming the crate to something like op-succinct-core-utils.